includes/class-wc-payments-features.php

```diff
@@ -119,6 +119,15 @@ public static function is_documents_section_enabled() {
 		return '1' === get_option( '_wcpay_feature_documents', $is_documents_enabled ? '1' : '0' );
 	}
 
+	/**
+	 * Checks whether custom deposit schedules are enabled.
+	 *
+	 * @return bool
+	 */
+	public static function is_custom_deposit_schedules_enabled() {
+		return '1' === get_option( '_wcpay_feature_custom_deposit_schedules', '0' );
```

Comment:
Hey @Jinksi, I think we should default to this feature flag being **on**, i.e. change the second param of `get_option` to `'1'`.

This will make the changes visible as merchants update to 4.7.

The other approach would be to simply remove the feature flag and related code, but this would be a fairly significant change and would require another review and additional testing.

It also means that merchants don't have the ability to "turn off" this functionality in the scenario that we ship a bug.

We can always remove the feature flag in a subsequent release.

What do you think?

Reply:
Sounds good! I'll update the flag to ensure it is enabled by default 👍

I'm testing the upgrade path 4.6.0 → 4.7.0 to ensure there are no hiccups with cached account data, so I'll test the feature flag is enabled by default also.

Reply:
Updated in 053fc0b and I'll update the PR title/desc now